test/e2e/specs/onboarding/signup__videopress-tailored.ts

```diff
@@ -49,6 +50,7 @@ describe( DataHelper.createSuiteTitle( 'Signup: Tailored VideoPress' ), () => {
 		} );
 
 		it( 'Enter account details', async function () {
+			page.setDefaultTimeout( 60 * 1000 );
```

Comment:
As above: We can be more explicit and extend the timeout for the `waitForURL` below.